JS/Lesson34/gulcin/recipe-app/src/modules/recipes/RecipeDetails/styles.css

```diff
@@ -0,0 +1,66 @@
+h2 {
```

Comment:
When using plain external css files in react, the styles defined in any of the files are applied everywhere in the app. Therefore, it's very dangerous to add generic styles like this to a styles file of some component. Instead, add them to the index.css files in the 'src' folder. 
Only add styles for classes that are specific to this particular component in the component styles file.